frontend/src/utils/DeckMapView.tsx

```diff
@@ -63,6 +73,9 @@ export const DeckMapView = ({ deck, markedRobotPosition }: DeckProps) => {
         return image
     }
 
+    let mapName = mapMetadata?.mapName.split('.')[0].replace(/[^0-9a-z-A-Z ]/g, ' ')
+    mapName = mapName ? mapName.charAt(0).toUpperCase() + mapName.slice(1) : ' '
```

Comment:
If we need to do this operation each time we get this data, might it be better to do it in getDeckMapMetadata? (Although I would personally prefer not doing this in the frontend, since it makes assumptions about the data sent from the backend/database)